samples/matter/lock/sample.yaml

```diff
@@ -2,54 +2,25 @@ sample:
   description: Matter Lock example
   name: Matter Lock
 tests:
-  sample.matter.lock.debug:
-    build_only: true
-    integration_platforms:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf7002dk_nrf5340_cpuapp
-    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
-    tags: ci_build
-  sample.matter.lock.no_dfu:
-    build_only: true
-    extra_args: CONF_FILE=prj_no_dfu.conf
-    integration_platforms:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf21540dk_nrf52840
-      - nrf7002dk_nrf5340_cpuapp
-    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf21540dk_nrf52840 nrf7002dk_nrf5340_cpuapp
-    tags: ci_build
   sample.matter.lock.release:
     build_only: true
     extra_args: CONF_FILE=prj_release.conf
     integration_platforms:
-      - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
-      - nrf7002dk_nrf5340_cpuapp
-    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
-    tags: ci_build
+    platform_allow: nrf52840dk_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
   sample.matter.lock.smp_dfu_ffs:
     build_only: true
-    extra_args: CONFIG_CHIP_DFU_OVER_BT_SMP=y CONFIG_CHIP_LIB_SHELL=y CONFIG_CHIP_COMMISSIONABLE_DEVICE_TYPE=y
+    extra_args: CONFIG_CHIP_LIB_SHELL=y CONFIG_CHIP_DFU_OVER_BT_SMP=y CONFIG_CHIP_COMMISSIONABLE_DEVICE_TYPE=y
       CONFIG_CHIP_ROTATING_DEVICE_ID=y CONFIG_CHIP_DEVICE_TYPE=10
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
-    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
-    tags: ci_build
-  sample.matter.lock.smp_dfu_ffs_wifi:
+    platform_allow: nrf52840dk_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
+  sample.matter.lock.wifi_smp_dfu_ffs_fd:
```

Comment:
I will move the factory data feature to a different example, due to fact that we would like to use this example for wifi integration as for nrf52840dk and nrf5340dk_cpuapp.
We will be able to get the wifi example prepared at twister stage for integration wifi tests

Reply:
OK factory data will be enabled in Weather Station, DL Wi-Fi will be the same as for other configurations for integration system tests.